xbmc/favourites/FavouritesService.cpp

```diff
@@ -12,17 +12,25 @@
 #include "ServiceBroker.h"
 #include "URL.h"
 #include "Util.h"
+#include "commons/ilog.h"
```

Comment:
This seems strange...

Reply:
Same comment as the other one. Use for the LOG_DEBUG, etc defines. Can map symbols like these to other headers (or nothing at all apparently) if there's something else you want it to handle them with. Just let me know